### PR TITLE
use pgbouncer 1.9

### PR DESCRIPTION
**Pull request by MichalPloski**

### CODE REVIEW

File: docker/k8s/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM alpine:latest
+
+MAINTAINER sre@shedul.com
```
Comment (medzin) [inline docker/k8s/Dockerfile:3]: Whe have sre alias? :trollface:

File: docker/k8s/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+RUN apk add --update python python-dev py-pip build-base postgresql-dev git && pip install pyyaml psycopg2 && rm -rf /var/cache/apk/*
+RUN git clone https://github.com/surgeventures/newrelic-plugin-agent.git src
```
Comment (medzin) [inline docker/k8s/Dockerfile]: Can you just add the sources from the working directory? Dockerfile is in the same repository.

**Review by medzin — CHANGES_REQUESTED** [review]

File: docker/k8s/Dockerfile (hunk 2 above)
Comment (MichalPloski) [reply docker/k8s/Dockerfile]: ok

File: docker/k8s/Dockerfile (hunk 1 above)
Comment (MichalPloski) [reply docker/k8s/Dockerfile:3]: why not? :-)

**Review by medzin — APPROVED** [review]

**Pull request merged**